sympy/stats/rv_interface.py

```diff
@@ -208,6 +208,31 @@ def skewness(X, condition=None, **kwargs):
     """
     return smoment(X, 3, condition, **kwargs)
 
+def kurtosis(X, condition=None, **kwargs):
+    """
+    Measure of the tailedness(outliers) of the probability distribution
+
+    Kurtosis of any univariate normal distribution is 3. Kurtosis less than
+    3 means that the distribution produces fewer and less extreme outliers
+    than the normal distribution
+
+    kurtosis(X) = E( ((X - E(X))/sigma)**4 )
```

Comment:
No space after ( and before ). Consider using math (LaTeX) formatting here: 
```
`\operatorname{Kurtosis}(x) = E\left[\left(\frac{X - E(X)}{\sigma}\right)^4\right]`  
```
(with backticks around it)